lib/composer.js

```diff
@@ -100,6 +101,10 @@ function generateJsonReport(
   return data;
 }
 
+function getDefaultProjectName(manifestPath) {
+  return path.dirname(manifestPath).split(path.sep).pop();
```

Comment:
you should use here `path.resolve` (here or above) so that you get the full path, otherwise if user runs `snyk test` inside the `php` project, this code will result in the name `'.'`